css/theme/black.css

```diff
@@ -176,6 +176,8 @@ body {
 .reveal table th,
 .reveal table td {
   text-align: left;
+  color: white;
+  font-size: 0.6em;
```

Comment:
do we really need those changes? If so, I would suggest to make them in a separate css file, without touching the css theme file of reveal.js